test/main/store/actions/index.test.js

```diff
@@ -984,6 +985,102 @@ describe('#setBlockHeight', () => {
   })
 })
 
+describe('#updateAccount', () => {
+  let main
+
+  const updaterFn = (node, id, update) => {
+    if (node === 'main.accounts') {
+      main.accounts[id] = update(main.accounts[id])
+    }
+
+    if (node === 'main.accountsMeta') {
+      main.accountsMeta[id] = update(main.accountsMeta[id])
+    }
+  }
+
+  beforeEach(() => {
+    jest.useFakeTimers() // somehow required despite being enabled globally
```

Comment:
@goosewobbler I think this was because some of the old code in the other tests were toggling back and forth between real and fake timers. I removed all of this and it looks to be working as expected now

Reply:
ah nice, will remove

Reply:
oh you did already, good